### DISCUSSION TITLE
I need a few pointers on how to get the krita plugin working for those already having a comfyui instaled.

### DISCUSSION THREAD

**Discussion by dadadies** [Acly/krita-ai-diffusion#704] [Q&A] [answered]
There are some files that are required but it doesnt tell me where to place many of them.  I know most of them go into the custom_node folder but some i don't now where they go. Also I cannot download 'Places_512_FullData_G.pth' it keeps giving me errors part way through the download.

**Comment by freecoderwaifu** [comment, marked as answer]
It's been a while, but after installing ComfyUI Manager you can install the custom nodes needed, directly from its built in search. 

https://github.com/Acly/krita-ai-diffusion/wiki/ComfyUI-Setup#requirements-for-comfyui

https://github.com/ltdrdata/ComfyUI-Manager?tab=readme-ov-file#installation

IP Adapter should be the Plus one, ComfyUI_IPAdapter_plus

Then the download script can download all the corresponding models. 

https://github.com/Acly/krita-ai-diffusion/wiki/ComfyUI-Setup#-download-script

The script should be in `AppData\Roaming\krita\pykrita\ai_diffusion`. You can open CMD or Terminal there to run it, just change the path to your ComfyUI install, and it should point to the actual ComfyUI folder if using the portable version. Something like this:

```
python -m pip install aiohttp tqdm
python download_models.py "E:\ComfyUI_windows_portable\ComfyUI\"
```

I think you can also download the models using ComfyUI Manager, or manually from the links in the wiki and place them in their respective folders.

**Comment by rbfbatx** [comment]
I don't know if its the right way but I just combined the server folder with the comfyui install folder.

**Comment by dadadies** [comment]
I have portable and am doing it all manually. Hopefully it works out. The manager doesnt work for me as my connection is too slow (under 1mb) so anything like pip, github, etc. always return a connection error. But Ill make it work.

**Comment by dadadies** [comment]
Took over 2 days to get everything setup because of the 1mb internet speed. It was a bit complicated, i think i had 2 projects to setup or this project had a sub project to setup. But the work is done.
